### ISSUE TITLE
Android build targets failing

### ISSUE THREAD

**Issue by tkidd77** [lucor/fyne-cross#28]
Any build target of Android fails with the following error. All 4 arch result in the same error. Windows, Linux, and Darwin are all successful. It seems like the lucor/fyne-cross:android container is missing some required packages.

```
Target android/amd64
No module found. Downloading dependencies using go get
Building...
# fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw
In file included from /go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/c_glfw.go:4:
In file included from ./glfw/src/context.c:28:
In file included from ./glfw/src/internal.h:169:
./glfw/src/x11_platform.h:36:10: fatal error: 'X11/Xlib.h' file not found
#include <X11/Xlib.h>
         ^~~~~~~~~~~~
1 error generated.
# fyne.io/fyne/vendor/github.com/go-gl/gl/v3.2-core/gl
/go/src/fyne.io/fyne/vendor/github.com/go-gl/gl/v3.2-core/gl/procaddr.go:58:11: fatal error: 'GL/glx.h' file not found
        #include <GL/glx.h>
                 ^~~~~~~~~~
1 error generated.
exit status 2
```

**To Reproduce**
```
fyne-cross --targets=android/amd64
```

**Comment by lucor** [comment]
@tkidd77 It seems you are using the stable version of Fyne.
To test android the develop branch is required (it will be available into the upcoming v1.2).
Could you please check the version of Fyne you are using?

**Comment by tkidd77** [comment]
Ah, I see. Yes, I am on the stable version of Fyne. I will wait for the 1.2 release. Thanks for the details!
